openPrompt/described_structure/readme.md

```diff
@@ -0,0 +1,30 @@
+## described_structure
```

Comment:
I think use snake_case in title is improper，will it be better to use "Title Case" for title?